### PR TITLE
Task 01 - setup

### PR DESCRIPTION
**Pull request by Bogdandrummer**
-  Mentor is added to pr
- AC copied to pr description
- link to deployed version added to pr
- About block (text and image) added to main page
- On the widths 390px and 1280px, the site layout corresponds to the design

Link to deployed page:
https://bogdandrummer.github.io/fe-react-2024/

### CODE REVIEW

File: src/components/about/About.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import './About.css';
+import '../../reset.css';
+
+function About() {
+    return (
+        <>
+            <HeaderComponent />
+            <main className="main__block">
+                <div className="main__row content">
+                    <div className="about__block">
```
Comment (MZabolotnev) [inline src/components/about/About.tsx]: It can be more semantic. Use article and section tags instead div.

File: src/components/about/About.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                                create my own small project for the portfolio.
+                            </p>
+                            <p className="about__text">
+                                You can contact me via
+                                <a
+                                    href="https://www.linkedin.com/in/%D0%B1%D0%BE%D0%B3%D0%B4%D0%B0%D0%BD-%D1%8F%D1%86%D0%B5%D0%BD%D1%82%D1%8E%D0%BA-841247270/"
+                                    className="about__link"
+                                >
+                                    LinkedIn profile
+                                </a>{' '}
```
Comment (MZabolotnev) [inline src/components/about/About.tsx]: {' '} - Is it necessary here?

File: src/reset.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
+@charset "UTF-8";
+/*Обнуление*/
```
Comment (MZabolotnev) [inline src/reset.css]: Please use English

File: src/reset.css [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+button::-moz-focus-inner {
+  padding: 0;
+  border: 0;
+}
+
+/*a {*/
+/*  text-decoration: none;*/
+/*}*/
```
Comment (MZabolotnev) [inline src/reset.css]: Pls remove commented code

**Comment by MZabolotnev** [comment]
Mobile view does not match design. Pls check font size, photo should be from above, and with paddings. Also no padding before profile link. 
image
image

**Review by MZabolotnev — CHANGES_REQUESTED** [review]

**Comment by vladyslav-yermolin-moc** [comment]
Hey, @Bogdandrummer, you have created pr to our main repo - https://github.com/MastersAcademy/fe-react-2024. 

But according to our flow, you should create pr to your repo - https://github.com/Bogdandrummer/fe-react-2024/tree/main

So I close this pr

**Pull request closed without merging**